service/src/test/java/uk/nhs/adaptors/gp2gp/utils/DateFormatUtilTest.java

```diff
@@ -64,6 +66,24 @@ public void When_FormattingDateTimeTypeToText_Expect_Hl7InUkZone(String input, S
         assertThat(actual).isEqualTo(expected);
     }
 
+    @ParameterizedTest
+    @MethodSource("convertedBackDates")
```

Comment:
In this case you could try `@CsvSource`. As `dateParams` is used twice though you'd need to decide if you want to repeat the data (which is fine for tests but nevertheless rubs some people the wrong way) or just vary the data a bit to keep things distinct.